src/api/package.json

```diff
@@ -31,6 +31,7 @@
     "@keystone-6/auth": "^7.0.3",
     "@keystone-6/core": "^5.8.0",
     "@keystone-6/fields-document": "^8.0.2",
+    "@paralleldrive/cuid2": "^2.2.2",
```

Comment:
is this package added to the ticket to remove after migration?

Reply:
Yes, EMS-3521